ext/standard/tests/array/is_countable.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+Check if variables can be counted
+--FILE--
+<?php
+
+$types = [
+    "array"     =>  [],
+    "null"      =>  null,
+    "string"    =>  "test",
+    "integer"   =>  123,
+    "float"     =>  3.14,
+    "true"      =>  true,
+    "false"     =>  false,
+    "object"    =>  (object) [],
+    "simplexml" =>  new SimpleXMLElement("<xml><tag1></tag1><tag2></tag2></xml>"),
```

Comment:
I know why it's using this class, but it's a bit strange to have a standard test bound to simplexml, which isn't always loaded.

Could we maybe find some other object with the same behaviour ?